common/text/macro_definition.h

```diff
@@ -85,6 +85,11 @@ class MacroDefinition {
 
   void SetDefinitionText(const TokenInfo& t) { definition_text_ = t; }
 
+  // Lexes the definition_text_ and store it in lexed_macro_body
+  void SetLexedDefinitionText(const std::vector<TokenInfo> l) {lexed_macro_body = l ;} 
```

Comment:
Don't pass vectors by value.

I'd probably call it `SetLexedDefinitionTokens()` to better reflect what we do. Also `lexed_macro_body` should be `lexed_definition_tokens_` then so that we don't have many names for the same thing.